### DISCUSSION TITLE
how the "expand" the graph

### DISCUSSION THREAD

**Discussion by krausdigital** [RomRider/apexcharts-card#794] [Q&A]
hello,

i have the problem, that there is a gap at the beginning and at the end of the chart.

any ideas?

![462914769_3918220325133107_6267955053293418766_n](https://github.com/user-attachments/assets/0329226e-00a9-4959-aa38-5ea8305f41c2)

**Comment by TheSerapher** [comment]
My guess is that you are not using the statistics feature. HA only keeps 10 days in recent data and everything else is moved into the recorder (statistics). You can add this to your series to fetch data from the statistics table (maybe adjust the `type` and `period` to what you need, check documentation):

```
  statistics:
    type: change
    period: day
```
